src/opm/output/eclipse/Summary.cpp

```diff
@@ -484,10 +484,15 @@ inline quantity glir( const fn_args& args ) {
 
     for (const auto& well : args.schedule_wells) {
         auto xwPos = args.wells.find(well.name());
-        if (xwPos != args.wells.end())
-            alq_rate += xwPos->second.rates.get(rt::alq, 0.0);
-    }
+        if (xwPos == args.wells.end())
+            continue;
 
+        if (well.isInjector())
+            continue;
```

Comment:
This check should be first to avoid having to perform expensive look-ups when the result is never going to be needed.